guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -9,6 +9,15 @@ The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+ifdef::foreman-el,foreman-deb[]
+To use the `ssh` mode, run the following command on your {ProjectServer}:
```

Comment:
```suggestion
To enable remote execution, run the following command on your {ProjectServer}:
```

"To use the ssh mode" implies that this command only allows use of the ssh mode, when actually it's required for both pull and ssh.